config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -579,10 +579,12 @@ presubmits:
   - name: pull-etcd-release-tests
     optional: true # remove once the job is stable
     cluster: k8s-infra-prow-build
-    always_run: true
+    always_run: false
```

Comment:
Question: eventually we still need to set it to `true`?

Reply:
Yes. I just added a comment to make it explicit.